apps/complete_registration/src/screens/dashboard/screen.tsx

```diff
@@ -48,6 +54,21 @@ export function DashboardScreen() {
         return <DashboardError />
     }
 
+    const formCards = Object.entries(forms)
+        .filter(([, formMeta]) => companyStatus == null || !FORM_HIDDEN_DURING[formMeta.key]?.includes(companyStatus))
+        .map(([key, formMeta]) => (
+            <FormCard
```

Comment:
Nitpick: I usually prefer having all react code and components in one place and the dataprocessing in another to help with separation, this also makes it easier to cache or memoize the data if required later. I would therefore just return the data from this codeblock and rename the variable to eg: formCardsData, then render the actual component in the tree